### ISSUE TITLE
Doc example screenshots don't match output from code

### ISSUE THREAD

**Issue by nikolausWest** [rerun-io/rerun#3848]
Here is an example for pinhole + depth image: https://www.rerun.io/docs/reference/types/archetypes/depth_image#depth-to-3d-example

Code
```python
"""Create and log a depth image and pinhole camera."""

import numpy as np
import rerun as rr

depth_image = 65535 * np.ones((8, 12), dtype=np.uint16)
depth_image[0:4, 0:6] = 20000
depth_image[4:8, 6:12] = 45000

rr.init("rerun_example_depth_image", spawn=True)

# If we log a pinhole camera model, the depth gets automatically back-projected to 3D
rr.log(
    "world/camera",
    rr.Pinhole(
        width=depth_image.shape[1],
        height=depth_image.shape[0],
        focal_length=20,
    ),
)

# Log the tensor.
rr.log("world/camera/depth", rr.DepthImage(depth_image, meter=10_000.0))

```

Image in docs
![image](https://github.com/rerun-io/rerun/assets/2624717/7754c9ab-80c0-43f3-bd3d-e6db1b26f852)

Reality
![image](https://github.com/rerun-io/rerun/assets/2624717/6dfa765a-14bc-400e-9e3c-814a8fe0d7ab)

(after manually setting "Backproject Radius Scale")
![image](https://github.com/rerun-io/rerun/assets/2624717/84d937e1-62d0-4bd3-acb4-8e4392fee717)

In this case, the image looked alot nicer!

**Comment by Wumpf** [comment]
looks like someone wanted to simplify the example at some point! Let's give the depth image a higher resolution and take the picture again

**Comment by teh-cmc** [comment]
Isn't that done @abey79 ?

**Comment by abey79** [comment]
Yes, fixed by #3954
